Doc/library/tkinter.font.rst

```diff
@@ -0,0 +1,93 @@
+:mod:`tkinter.font` --- Tkinter font wrapper
+============================================
+
+.. module:: tkinter.font
+   :platform: Tk
+   :synopsis: Tkinter font-wrapping class
+
+**Source code:** :source:`Lib/tkinter/font.py`
+
+--------------
+
+The :mod:`tkinter.font` module provides the :class:`Font` class for creating
+and using named fonts.
+
+The different font weights and slants are:
+
+.. data:: NORMAL
+          BOLD
+          ITALIC
+          ROMAN
+
+.. class:: Font(root=None, font=None, name=None, exists=False, options)
+
+   The :class:`Font` class represents a named font. *Font* instances are given
+   unique names and can be specified by their family, size, and style
+   configuration. Named fonts are Tk's method of creating and identifying
+   fonts as a single object, rather than specifying a font by its attributes
+   with each occurrence.
+
+    keyword arguments:
+
+       | *font* - font specifier tuple (family, size, options)
+       | *name* - unique font name
+       | *exists* - self points to existing named font if true
+
+    allowed keyword options (ignored if *font* is specified):
```

Comment:
How are these “keyword options” specified? Is there any relationship with the _options_ parameter?

Reply:
FYI in a couple Python programs I wrote, I pass the first five options as Python keyword parameters to the Font constructor, so maybe these are all meant to be extra parameters to the constructor, not related to _options_.

I always passed _size_ as a negative number. I vaguely remember that uses different units (pixels instead of points perhaps?)

Reply:
The *options* parameter is a variable-length list of keyword parameters (**kwargs in essence). So they're not actually extra parameters.

Yes, you are right - "If the size argument is a positive number, it is interpreted as a size in points. If size is a negative number, its absolute value is interpreted as a size in pixels" - Tcl/Tk docs.